pkg/util/utils.go

```diff
@@ -37,6 +37,13 @@ import (
 	"k8s.io/minikube/pkg/version"
 )
 
+type RetryableError struct {
+	Err error
+}
+
+func (r *RetryableError) Temporary() bool { return true }
```

Comment:
I think we can skip this temporary interface and method. Type assertion should work fine. The net example you sent actually has some logic in the temporary method, it doesn't always return true.

This also shows that type assertion works: https://play.golang.org/p/3KZAAoqMTj